.github/autobuild/get_build_vars.py

```diff
@@ -43,7 +43,8 @@ def get_build_version(jamulus_pro_version):
 
 def set_github_variable(varname, varval):
     print("{}='{}'".format(varname, varval))  # console output
-    print("::set-output name={}::{}".format(varname, varval))
+    os.environ['GITHUB_OUTPUT'] = f"varname=varval"
```

Comment:
```suggestion
    os.environ['GITHUB_OUTPUT'] = f"varname=varval"
```

This sounds wrong, similar to the powershell part. The previous attempt was better, but lacked a new line, I think.